### ISSUE TITLE
change message for 'Publish' button to reflect that ther publishing request will be put in a queue

### ISSUE THREAD

**Issue by heshamkelbashandy** [heshamkelbashandy/ess-dive-catalog#107]
In GitLab by @vchendrix on Mar 12, 2018, 11:59

change message for 'Publish' button to reflect that ther publishing request will be put in a queue
```
Are you sure you want to publish? Publishing the data package will make it publicly available for search and download.

The data package will be published with an existing DOI (if applicable), or else a new DOI will be issued through ESS-DIVE.  You will be contacted by an ESS-DIVE admin with further details of your request.
```

![Screen_Shot_2018-03-12_at_11.58.19_AM](/uploads/cb1092d1d3722cf333f7d419f7424e03/Screen_Shot_2018-03-12_at_11.58.19_AM.png)

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Mar 12, 2018, 15:06

added ~3203215 label

**Comment by heshamkelbashandy** [comment]
In GitLab by @charuleka on Mar 15, 2018, 11:09

Are you sure you want to Publish? Publishing the data package will make it publicly available for search and download.

The data package will be published with an existing DOI (if applicable), or else a new DOI will be issued through ESS-DIVE.  

If you click "OK", this data package will be put into a publication queue. An ESS-DIVE admin will check the submission and contact you through the email address in your profile. The data package will not be published until you respond to our email and address all issues regarding your submission.

**Comment by heshamkelbashandy** [comment]
In GitLab by @charuleka on Mar 15, 2018, 11:10

assigned to @elbashandy and unassigned @charuleka

**Comment by heshamkelbashandy** [comment]
In GitLab by @charuleka on Mar 15, 2018, 11:10

added ~3248326 label

**Comment by heshamkelbashandy** [comment]
In GitLab by @elbashandy on Mar 19, 2018, 12:40

mentioned in merge request !42

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Mar 19, 2018, 14:55

closed via commit dc5fd40e2b702c5db12936d53c03d2fc5c7f332b

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Mar 19, 2018, 14:55

closed via merge request !42
